03_Modules/Credentials/src/module/api.ts

```diff
@@ -162,6 +162,7 @@ export class CredentialsModuleApi
    * and register token B and C.
    */
   @Post('/register-credentials-token-a')
+  @AsAdminEndpoint()
```

Comment:
Nice catch! Would you mind helping add this annotation for `@Post('/generate-credentials-token-a')` and `@Put('/regenerate-credentials-token')` as well?

Reply:
added